src/Util/DependencyPackages.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Created by PhpStorm.
+ * User: Daler B. (@bdaler)
+ * Date: 08.02.2023
+ * Time: 21:18.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+class DependencyPackages

```

Comment:
"I just thought, maybe it could be a new enum in PHP 8.1?"

Reply:
this would be a bad usage of an enum. Please using those constants need a string.

Btw, I'm not even sure those constants provide much benefit.

Reply:
@stof Hi. The essence of using constants is that, for example, the `orm` package is used in 4 or 5 files. In the future (of course, this is unlikely, but nevertheless), if it is necessary to change this, it will have to be done in 4 or 5 different places. But with constants, you can make the change in one place and it will apply to all.

Reply:
Really Enum is not a variant, however class is not a good way too I think, it will be better to use `Interface` as constant keeper, and also it will be better to keep original package names (aliases used in code) not real packages names.